app/models.py

```diff
@@ -372,7 +372,7 @@ class SystemProfileSchema(MarshmallowSchema):
     bios_version = fields.Str(validate=marshmallow_validate.Length(max=100))
     bios_release_date = fields.Str(validate=marshmallow_validate.Length(max=50))
     cpu_flags = fields.List(fields.Str(validate=marshmallow_validate.Length(max=30)))
-    operating_system = fields.List(fields.Nested(OperatingSystemSchema()))
+    operating_system = fields.Nested(OperatingSystemSchema)
```

Comment:
I think this should be 
```suggestion
    operating_system = fields.Nested(OperatingSystemSchema())
```

Reply:
I'm actually not clear on this... [the docs](https://marshmallow.readthedocs.io/en/3.0/nesting.html) make it look like you don't use the parens, but some examples use parens when supplying a param

Reply:
The use of parens is a little inconsistent elsewhere in this file, but for this section it seems to always pass "()", so might be better to include them.

@CodeHeeler @tahmidefaz Are there sufficient existing tests for this change?

Reply:
I pushed the parentheses change; we can revert this if we decide against it later